### PR TITLE
move components to a plugin

### PR DESCRIPTION
**Pull request by brandonreid**
I was having issues where the custom component classes weren't available in the demo files. Importing this lib into the vue-charts repo, I also had issues where the build wasn't recognizing the custom component classes.

Looking in this repo, I saw that the `forms` plugin was imported twice, and presume the forms weren't being imported from out tailwindPlugins file, but were directly. 

This change fixes my issues, feels a little cleaner as well.

### CODE REVIEW

**Comment by pleek91** [comment]
@brandonreid I'd prefer if we could combine the theme and utilities plugins into a single plugin. We're unlikely to ever install one and not the other so its a simpler install with a single plugin file.

**Comment by brandonreid** [comment]
> @brandonreid I'd prefer if we could combine the theme and utilities plugins into a single plugin. We're unlikely to ever install one and not the other so its a simpler install with a single plugin file.

Man I was struggling to make this happen this morning, but it clicked now. Combined into one file.

File: public/tailwindPlugin.js [hunk 1]
```diff
@@ -1,7 +1,6 @@
 /* eslint-disable @typescript-eslint/explicit-function-return-type */
 /* eslint-disable @typescript-eslint/no-require-imports */
 /* eslint-disable @typescript-eslint/no-var-requires */
-const forms = require('@tailwindcss/forms')
```
Comment (pleek91) [inline public/tailwindPlugin.js:4]: I'm not quite understanding why we're removing the forms plugin from the prefect-design plugin. Wont this have negative effects assuming we're depending on the forms plugin always being installed whenever the prefect-design plugin is used?

Comment (pleek91) [reply public/tailwindPlugin.js:4]: I thought forms was added to the config? That's what I'm seeing locally which appears to be valid looking at the types.

Comment (brandonreid) [reply public/tailwindPlugin.js:4]: It is, yeah. Forms was previously added twice. Once in the tailwindPlugin file, where it was the third argument and I don't think it was actually doing anything, then once in the tailwind.config file, where it was actually being imported and subsequently bundled with the build.

Comment (pleek91) [reply public/tailwindPlugin.js:4]: Okay, trust you're correct 👍

**Pull request merged**